src/ClusterBootstrap/params.py

```diff
@@ -813,6 +822,8 @@
     "vc_config": {
         "VC-Default": ["*"],
     },
+    "ost_reboot_wait_time": "30s",
+    "storage_quota": {"user_soft": "2G", "user_hard": "3G", "user_grace_period": "1w"}, 
```

Comment:
We should support not setting storage quota by default.

Reply:
ok, sure, then I'd just remove this line.